enterprise/ha/src/test/java/org/neo4j/kernel/ha/transaction/TransactionThroughMasterSwitchStressIT.java

```diff
@@ -77,27 +81,38 @@
  * This test is a stress test and duration of execution can be controlled via system property
  * -D{@link org.neo4j.kernel.ha.transaction.TransactionThroughMasterSwitchStressIT}.duration
  */
+@RunWith( Parameterized.class )
 public class TransactionThroughMasterSwitchStressIT
 {
+    @Parameter
+    public boolean txTerminationAwareLocks;
+
     @Rule
     public final ClusterRule clusterRule = new ClusterRule( getClass() )
-                .withInstanceSetting( HaSettings.slave_only,
-                        new IntFunction<String>() // instances 1 and 2 are slave only
+            .withSharedSetting( tx_termination_aware_locks, String.valueOf( txTerminationAwareLocks ) )
```

Comment:
Yeah, this is a mistake. Fixed here https://github.com/neo4j/neo4j/commit/fcedeec9c1229fd55c2636e0d705b68a03e97798. Thanks!